kiwi/tasks/image_info.py

```diff
@@ -101,7 +101,17 @@ def process(self):
             solver = self._setup_solver()
             package_list = self.xml_state.get_bootstrap_packages() + \
                 self.xml_state.get_system_packages()
-            solved_packages = solver.solve(package_list)
+            bootstrap_packages = solver.solve(
+                self.xml_state.get_bootstrap_packages(), False,
+                True if self.xml_state.get_bootstrap_collection_type() is
+                'onlyRequired' else False
+            )
+            solved_packages = solver.solve(
+                self.xml_state.get_system_packages(), False,
+                True if self.xml_state.get_system_collection_type() is
+                'onlyRequired' else False
+            )
+            solved_packages.update(bootstrap_packages)
```

Comment:
@schaefi damm... your doubts pointed to the right direction... there are little differences within resolving bootstrap and image package all at once or in separate calls. Very little differences, but differences after all.

I used Leap 42.2 JeOS for the tests, and there were about 4 or 5 different libraries. In that case I think that or we find a way to make cumulative calls (at first sight I did not see how) or we assume a common configuration for bootstrap and image packages.